### DISCUSSION TITLE
Does This Library work in Swift 5.7?

### DISCUSSION THREAD

**Discussion by ajdali** [vyshane/grpc-swift-combine#29] [General]
Hi there, I had some issues importing the library with swift upgraded version. i was unable to import with the latest version of the swift-grpc library.

**Comment by vyshane** [comment]
Hi! The library works with Swift 5.7.

Version [1.0.13](https://github.com/vyshane/grpc-swift-combine/releases/tag/1.0.13) requires Swift gRPC 1.9.0, not their latest version.

If you want to use the latest version of Swift gRPC (1.13.0), I have just released version [1.1.0](https://github.com/vyshane/grpc-swift-combine/releases/tag/1.1.0), which depends on Swift gRPC 1.13.0.

Hope that helps!
